dev/tools/controllerbuilder/cmd/root.go

```diff
@@ -43,11 +43,7 @@ func buildAddCommand(baseOptions *options.GenerateOptions) *cobra.Command {
 				Kind:        kind,
 				KindToLower: strings.ToLower(kind),
 			}
-			path, err := scaffold.BuildControllerPath(baseOptions.ServiceName, kind)
-			if err != nil {
-				return err
-			}
-			return scaffold.Scaffold(path, cArgs)
+			return scaffold.Scaffold(baseOptions.ServiceName, kind, cArgs)
```

Comment:
Nit: I think we should just pass cArgs, as the first two args are already in cArgs. (it would get confusing if someone passed a different value for cArgs.service vs the first argument)